### PR TITLE
omxplayer: remove hardcoded tune and arch from Makefile

### PR DESCRIPTION
**Pull request by c0ffee**
Fixes: #428

Signed-off-by: Martin Schuessler <tossprobe@gmail.com>

**- What I did**
Backported the fix for compiling recent omxplayer with thumb support
**- How I did it**

### CODE REVIEW

**Comment by agherzan** [comment]
@jenkins-ag ok to test

**Comment by agherzan** [comment]
Thanks @c0ffee . Merged to warrior.

File: recipes-multimedia/omxplayer/omxplayer/0007-Remove-Makefile-hardcoded-arch-tune.patch [hunk 1]
```diff
@@ -0,0 +1,8 @@
+--- a/Makefile	2019-06-20 15:04:53.390282996 +0200
++++ b/Makefile	2019-06-20 15:03:45.538763872 +0200
+@@ -1,4 +1,4 @@
+-CFLAGS=-pipe -mfloat-abi=hard -mcpu=arm1176jzf-s -fomit-frame-pointer -mabi=aapcs-linux -mtune=arm1176jzf-s -mfpu=vfp -Wno-psabi -mno-apcs-stack-check -g -mstructure-size-boundary=32 -mno-sched-prolog
++CFLAGS+= -fomit-frame-pointer -mabi=aapcs-linux -Wno-psabi -mno-apcs-stack-check -g -mstructure-size-boundary=32 -mno-sched-prolog
```
Comment (kraj) [inline recipes-multimedia/omxplayer/omxplayer/0007-Remove-Makefile-hardcoded-arch-tune.patch]: some of these options are gcc specific and now they will get added for clang as well. I wonder if we can remove these options if we can ?

Comment (agherzan) [reply recipes-multimedia/omxplayer/omxplayer/0007-Remove-Makefile-hardcoded-arch-tune.patch]: It would need some checking.

**Pull request merged**